packages/css/src/radio-button/README.md

```diff
@@ -0,0 +1,6 @@
+# Radio button
```

Comment:
General question: do we want to offer a radio button like this, without a label? You would never use it without a label right?